pandas/tests/test_common.py

```diff
@@ -817,6 +817,21 @@ def test_dict_compat():
     assert (com._dict_compat(data_unchanged) == data_unchanged)
 
 
+def test_is_timedelta():
+    assert (com.is_timedelta64_dtype('timedelta64'))
```

Comment:
find here for now. we need to add coverage for all of the ``is_*`` routines. I have a PR which moves all of this to ``pandas/types``, but ok for now